src/main/kotlin/com/openlattice/auditing/AuditInitializationTask.kt

```diff
@@ -56,7 +57,8 @@ class AuditInitializationTask(
                 UsersAndRolesInitializationTask::class.java,
                 OrganizationsInitializationTask::class.java,
                 Auth0SyncInitializationTask::class.java,
-                EdmSyncInitializerTask::class.java
+                EdmSyncInitializerTask::class.java,
+                IdConstantsReservationTask::class.java
```

Comment:
IdConstantsReservationTask is running already after EdmSyncInitializerTask, so I think  EdmSyncInitializerTask can be removed (unless we want list all for clarity)

Reply:
probably safe to remove but there seems to be some badness happening with lazy evaluation that we should look into at some point so I'd rather leave it for now just to be safe

Reply:
*evaluation of task order

Reply:
Ah, okay! 👍